sdk/monitor/azquery/version.go

```diff
@@ -8,5 +8,5 @@ package azquery
 
 const (
 	moduleName = "azquery"
-	version    = "v1.1.0-beta.2"
+	version    = "v1.1.0"
```

Comment:
This is a patch because it adds no new features or API
```suggestion
	version    = "v1.0.1"
```

Reply:
Bumped the number to 1.1.0 because we're releasing to stable the new QueryResource feature tomorrow. Just included the version change here so I didn't have to do a separate PR.